app/components/bookings/create/create-booking.component.html

```diff
@@ -0,0 +1,39 @@
+<ion-header>
+  <ion-navbar>
+    <ion-title>{{ _workpackage.stringId }} - {{ _workpackage.name }}</ion-title>
+  </ion-navbar>
+</ion-header>
+
+<ion-content>
+    <button accent floating block full *ngIf="!_bookingSession" (click)="createBookingSession()" >
+      <ion-icon name="ios-stopwatch-outline"></ion-icon>Livebuchung starten
+    </button>
+    <button accent floating block full *ngIf="_bookingSession" (click)="checkoutBookingSession()" >
```

Comment:
There seems to be an misunderstanding from the mockup!
In case the `BookingSession` is set, "Checkout" will only be shown as a header and not as a Button. The Booking is still created by pressing "Create Booking"! There would be no point in having two Buttons for a single functionality.